.github/workflows/upgrade-sdk.yml

```diff
@@ -0,0 +1,44 @@
+name: Update @cord-sdk/react package
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *' # Run every hour
+
+jobs:
+  update-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          # checkout pull request HEAD commit instead of merge commit
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install npm dependencies
+        run: npm install
+
+      - name: Check for new version
+        id: check-version
+        run: npx -p npm-check-updates ncu "/^@cord-sdk.*/" -u
+
+      - name: Install npm dependencies
+        run: npm install
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.CORD_OPS_USER_PAT }}
+          title: 'chore: Update cord sdk to newest version'
```

Comment:
Can we call this `Auto generated` or `Github generated` or something?  I don't personally love `chore`

Reply:
I guess since the author is the ops bot, we can drop it altogether